addons/l10n_fr_pos_cert/models/pos.py

```diff
@@ -41,7 +38,8 @@ def open_frontend_cb(self):
         return super(pos_session, self).open_frontend_cb()
 
 
-ORDER_FIELDS = ['date_order', 'user_id', 'lines', 'payment_ids', 'pricelist_id', 'partner_id', 'session_id', 'pos_reference', 'sale_journal', 'fiscal_position_id']
+ORDER_FIELDS_BEFORE_17_4 = ['date_order', 'user_id', 'lines', 'payment_ids', 'pricelist_id', 'session_id', 'pos_reference', 'sale_journal', 'fiscal_position_id', 'partner_id']
+ORDER_FIELDS_FROM_17_4 = ['date_order', 'user_id', 'lines', 'payment_ids', 'pricelist_id', 'session_id', 'pos_reference', 'sale_journal', 'fiscal_position_id', 'pos_version']
```

Comment:
With such change, a DB migrated from 17 to 18 will have hash control broken. and if we edit the document that certify that there is no change, the document will say that the orders are corrupted. 
is it deliberate ?